### ISSUE TITLE
Exploit fails to trigger on a console upgraded to 8MB

### ISSUE THREAD

**Issue by OldBoredEE** [brad-lin/FreePSXBoot#106]
This is pretty unusual case, but I thought it was worth reporting it in case anyone else runs into it.

Console is a SCPH-5501 (Boot ROM CRC = 8d8cb7e4) that's been expanded to have 8MB of main memory - on this console, going to the memory card screen with the FreePSXBoot memory card installed does nothing - it just displays the card as being blank.  The card is working normally on another SCPH-5501 with the same boot ROM, but the original stock 2MB RAM.  I guess something the exploit is relying on memory aliasing.

**Comment by nicolasnoble** [comment]
Yes, some versions of the exploit explicitly relies on the two address bits missing to roll over into kernel space. This indeed won't work on a modified console for which the bios requires this, as it'll hit upper ram instead of the kernel space.

**Comment by JMaxxL** [comment]
What kinda of things are possible besides developing games w/8MB ram?

**Comment by OldBoredEE** [comment]
> Is this the Asia VCD console ?

No, it's regular SCPH-5501 retail NTSC:U/C unit - it's just had the RAM chips replaced with larger ones to expand the total memory to 8MB to match the memory on the development boards.

The SCPH-5903 has the same 2MB RAM as other retail consoles.

**Comment by ab310714gz4700** [comment]
freepsxboot for 5903 is missing
